src/utils/format-url-path.ts

```diff
@@ -6,7 +6,8 @@
 export const getURLPath = (url: string | undefined) => {
     if (typeof url !== 'string') return url;
     if (url.startsWith('#')) return url;
-
+    if (url.indexOf('http://') === 0 || url.indexOf('https://') === 0)
```

Comment:
this is failing for links which are absolute urls. No change needed for absolute urls.

Reply:
I didn't intend for this to be for absolute URLs, but if this is needed for certain cases, then this change makes sense.